core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -366,7 +367,7 @@ private TestgridYaml buildTestgridYamlContent(JobConfigFile jobConfigFile) {
                     TestGridConstants.TESTGRID_YAML, " is missing in ", deployRepositoryLocation));
         }
         testgridYamlContent = testgridYamlBuilder.toString().trim();
-        if (testgridYamlContent.isEmpty()) {
+        if (testgridYamlContent.isEmpty() || !testgridYamlContent.contains("scenarioConfig")) {
```

Comment:
Seems we need scenarioConfig in lot of places. Shall we introduce a constant?

Reply:
+1

Reply:
Since this is being used only once do we need a constant?

Reply:
Then should be fine.